app/resources/api/v1/job_resource.rb

```diff
@@ -9,6 +9,9 @@ class JobResource < JSONAPI::Resource
 
       paginator :paged
 
+      belongs_to :work_order
+      belongs_to :work_plan
```

Comment:
Dont think the job belongs to a work_plan, it just belongs to a work_order, which belongs to a work_plan?

Reply:
That's what I thought too, but this seems to only way to be able to relate both resources to a job